docs/rst/user_guide/conventions.rst

```diff
@@ -16,28 +16,50 @@ Local coordinate definition
 Internally, Eradiate and its radiometric kernel Mitsuba use Cartesian
 coordinates to locate scene objects. Scene construction, when relevant, map the
 X, Y and Z axes to the local East, North and up directions at the origin
-location. Note that this does not mean that Eradiate always uses an East, North,
+location.
+
+.. only:: not latex
+
+   .. image:: fig/cartesian-coordinate-system.svg
+
+Note that this does not mean that Eradiate always uses an East, North,
 up (ENU) coordinate system: this is only relevant when planetary curvature can
 be neglected.
 
 Local illumination and viewing angle definition
 -----------------------------------------------
 
-Internally, Eradiate defines the zenith angle :math:`\theta` as the angular
-distance (in degree or radian) to the local zenith. From this, it follows that:
+Internally, Eradiate also uses the spherical coordinate system with the
+`ISO 80000-2:2019 convention <https://www.iso.org/standard/64973.html>`_
+(commonly used in physics) where:
+
+* :math:`r`, denotes the radial distance whose magnitude is a positive number,
+* :math:`\theta \in [0, \pi]` rad, equivalently :math:`[0, 180]°`, denotes the
+  zenith angle, a.k.a. the colatitude, polar angle, normal angle or
+  inclination angle, which measures the angular distance to the local zenith,
+* :math:`\varphi \in [0, 2\pi[` rad, equivalently :math:`[0, 360[°`, denotes
+  the azimuth angle.
+
+.. only:: not latex
+
+   .. image:: fig/spherical-coordinate-system.svg
```

Comment:
For now, most figures should be also exported to PNG for the PDF version as well. We also store them in the `docs/fig` directory (this will change in the future). Ditto for the other figures.